src/statement-list/statement_list_parser.cpp

```diff
@@ -97,6 +103,21 @@ float statement_list_parsert::find_version(const exprt &root)
   UNREACHABLE; // Root expression should always have a version
 }
 
+exprt statement_list_parsert::find_return_value(const exprt &root)
```

Comment:
Make `static` functions which don't require access to any `static` class fields free functions. For these functions here, this means you can remove them from the header, which increases encapsulation.